packages/firebase_vertexai/firebase_vertexai/lib/src/vertex_error.dart

```diff
@@ -44,6 +44,37 @@ final class UnsupportedUserLocation implements VertexAIException {
   String get message => _message;
 }
 
+/// Exception thrown when the service api is not enabled.
```

Comment:
[nit] capitalize acronyms like API in prose.

```suggestion
/// Exception thrown when the service API is not enabled.
```